src/main/groovy/org/shipkit/internal/gradle/CloneGitRepositoryTask.java

```diff
@@ -25,37 +31,92 @@
 
     private static final Logger LOG = Logging.getLogger(CloneGitRepositoryTask.class);
 
-    private String repository;
+    private String repositoryUrl;
     private File targetDir;
+    private int numberOfCommitsToClone;
```

Comment:
I suggest to call it explicitly 'cloneDepth' and link in the documentation to 'git clone --depth'